doc/api/buffer.markdown

```diff
@@ -1208,7 +1208,7 @@ buf.readUIntBE(0, 6).toString(16);
 Setting `noAssert` to `true` skips validation of the `offset`. This allows the
 `offset` to be beyond the end of the Buffer.
 
-### buf.slice([start[, end]])
+### buf.slice([start][, end])
```

Comment:
This is a good example of [what I was referring to](https://github.com/nodejs/node/issues/5971#issuecomment-203906563). It's not possible to pass *just* `end`, you have to also pass `start` if you want to specify `end`. The previous documentation style showed this dependency, but the new change makes it seem like you can perhaps pass `undefined` or something for `start` in order to just set `end`. It also adds ambiguity since if you just pass one numeric value, which parameter will it get treated as, since both are optional?